pandas/core/strings.py

```diff
@@ -1635,7 +1637,8 @@ def rsplit(self, pat=None, n=-1, expand=False):
     Examples
     --------
 
-    >>> s = Series(['A_B_C', 'D_E_F', 'X'])
+    >>> s = pd.Series(['A_B_C', 'D_E_F', 'X'])
```

Comment:
This docstring is shared with the Index. Could you add a short example with `Index` as well? (can be the same data as this one, just in `Index` instead of Series.